lib/infinum_json_api_setup/rspec/matchers/include_all_resource_ids.rb

```diff
@@ -38,17 +38,26 @@ def failure_message
         attr_reader :response
         attr_reader :required_ids
 
+        def process_required_ids(ids)
+          ids.sort
+        end
+
         def actual_ids
-          @actual_ids ||= JSON
-                          .parse(response.body)
-                          .fetch('data')
-                          .map { |resource| process_id(resource['id']) }
-                          .sort
+          @actual_ids ||= process_actual_ids(
+            JSON
+            .parse(response.body)
```

Comment:
Didn't we extract parts like these in the other PR?

Reply:
Yep I did, later #38 overhauled this implementation and introduced (a more) agnostic JSON body matching process. And also refactored this whole implementation